source/geometry_model/two_merged_boxes.cc

```diff
@@ -335,6 +336,26 @@ namespace aspect
 
 
 
+    template <int dim>
+    void
+    TwoMergedBoxes<dim>::make_periodicity_constraints(AffineConstraints<double> &constraints) const
+    {
+      using periodic_boundary_set
```

Comment:
Does this mean that any class that supports periodic boundaries now needs to copy this implementation? I don't think this is great as this implementation could be the default implementation in the Interface class. If rotation is needed like in the spherical shell, it can be overriden.